homeassistant/components/climate/nuheat.py

```diff
@@ -0,0 +1,212 @@
+"""
+Support for NuHeat thermostats.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/climate.nuheat/
+"""
+import logging
+from datetime import timedelta
+
+from homeassistant.components.climate import (
+    ClimateDevice,
+    STATE_HEAT,
+    STATE_IDLE)
+from homeassistant.components.nuheat import DATA_NUHEAT
+from homeassistant.const import (
+    ATTR_TEMPERATURE,
+    TEMP_CELSIUS,
+    TEMP_FAHRENHEIT)
+from homeassistant.util import Throttle
+
+DEPENDENCIES = ["nuheat"]
+
+_LOGGER = logging.getLogger(__name__)
+
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=5)
+
+MODE_AUTO = "auto"  # Run device schedule
```

Comment:
I think you should use the STATE_* constants from `climate/__init__.py` here.

Reply:
Yea, I found that some of these values existed in `const.py` and `climate/__init__.py`, but not all of them... so my thought was that defining them all here would be more straight forward as to what they are. I can make that change though.

Even states mentioned in the actual documentation (i.e. the `temperature` hold mode) are not defined in either `const.py` or `climate/__init__.py`

Here are the 2 that exist as states...
```
const.STATE_HOME = 'home'
climate.STATE_AUTO = 'auto'
```

I could import those and add my 2 new states, e.g.

```
from homeassistant.components.climate import STATE_AUTO
from homeassistant.const import STATE_HOME
...
STATE_HOLD_TEMPERATURE = "temperature"
STATE_TEMPORARY_HOLD = "temporary_temperature"
```

Reply:
I think `STATE_HOME` is only being used with the device / presence trackers to indicate that you / your device is in the home zone. I don't think we should overload the semantics here with a meaning in the climate device.

Reply:
One more thing: How does setting the operation mode to `STATE_AWAY` differ from setting the away mode to `on`? Sorry, I'm trying to wrap my head around this.